app/src/main/java/com/rahobbs/todo/fragments/DatePickerFragment.java

```diff
@@ -0,0 +1,79 @@
+package com.rahobbs.todo.fragments;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.DatePicker;
+
+import com.rahobbs.todo.R;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+/**
+ * Created by rachael on 2/5/16.
+ * Dialog fragment to control date picker
+ */
+public class DatePickerFragment extends DialogFragment {
```

Comment:
Hmmm the more I look at this class, I'm wondering if you even need it at all. Can you just use a `DatePicker` in the calling code?